File: CS/Models/GridModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Example.Models {
    public class GridModel {
        public Int32 Id { get; set; }
        public String Name { get; set; }

        public static List<GridModel> GetModels() {
            return new List<GridModel>() {
                new GridModel() { Id = 1, Name = "Mike" },
                new GridModel() { Id = 2, Name = "Serge" },
                new GridModel() { Id = 3, Name = "Kate" },
                new GridModel() { Id = 4, Name = "Roma R." },
                new GridModel() { Id = 5, Name = "Vest" },
                new GridModel() { Id = 6, Name = "Marion" }
            };
        }
    }
}

File: VB/Models/GridModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace Example.Models
	Public Class GridModel
		Private privateId As Int32
		Public Property Id() As Int32
			Get
				Return privateId
			End Get
			Set(ByVal value As Int32)
				privateId = value
			End Set
		End Property
		Private privateName As String
		Public Property Name() As String
			Get
				Return privateName
			End Get
			Set(ByVal value As String)
				privateName = value
			End Set
		End Property

		Public Shared Function GetModels() As List(Of GridModel)
			Return New List(Of GridModel) (New GridModel() {New GridModel() With {.Id = 1, .Name = "Mike"}, New GridModel() With {.Id = 2, .Name = "Serge"}, New GridModel() With {.Id = 3, .Name = "Kate"}, New GridModel() With {.Id = 4, .Name = "Vest"}, New GridModel() With {.Id = 4, .Name = "Marion"}})
		End Function
	End Class
End Namespace